homeassistant/components/light/zwave.py

```diff
@@ -97,11 +100,10 @@ def __init__(self, values, refresh, delay):
                     self._zw098 = 1
 
         # Used for value change event handling
-        self._refreshing = False
-        self._timer = None
         _LOGGER.debug('self._refreshing=%s self.delay=%s',
                       self._refresh_value, self._delay)
         self.value_added()
+        self.values.primary.set_change_verified(refresh)
```

Comment:
Since this is now using OZW `set_change_verified` and not dimmer-specific logic, maybe we should pull this from light-only to generic attribute?